app/src/main/java/com/mobiledevelopmentfoodapp/dto/Customer.kt

```diff
@@ -1,7 +1,7 @@
 package com.mobiledevelopmentfoodapp.dto
 
-//import androidx.room.Entity
-//import androidx.room.PrimaryKey
+// import androidx.room.Entity
+// import androidx.room.PrimaryKey
```

Comment:
changed formatting of existing comments within the project to be in line with proper conventions